src/OrbitGl/GlCanvas.h

```diff
@@ -73,18 +72,14 @@ class GlCanvas {
 
   [[nodiscard]] TextRenderer& GetTextRenderer() { return text_renderer_; }
 
-  // TODO: Only needed for Graph labels
+  // Only needed for Graph labels - should we find a better way?
```

Comment:
We should create a bug for that

Reply:
I've actually removed this, the method now exposes the last mouse pos (x and y), that makes it a bit more generically useful.